pages/api/credit-card/WordBank.ts

```diff
@@ -0,0 +1,5 @@
+export interface WordBank {
```

Comment:
do we need this? what's this for?

Reply:
better to have the type in the file that's used in

Reply:
It was used in the Drag n' Drop component so i'll remove this